lib/autoload/coursemology.rb

```diff
@@ -0,0 +1,7 @@
+module Coursemology
+  class ComponentNotFoundError < StandardError
```

Comment:
Then what's your suggestion ? I think all custom exceptions should be scoped under a file or a folder